tests/Auth0.AuthenticationApi.IntegrationTests/Tokens/IdTokenValidatorIntegrationTests.cs

```diff
@@ -77,8 +77,10 @@ public async Task Passes_Token_Validation()
 
             });
 
-            var idTokenValidation = new IdTokenRequirements($"https://{GetVariable("AUTH0_AUTHENTICATION_API_URL")}/", GetVariable("AUTH0_CLIENT_ID"), TimeSpan.FromMinutes(1));
-            await idTokenValidation.AssertTokenMeetsRequirements(authenticationResponse.IdToken);
+            var issuer = $"https://{GetVariable("AUTH0_AUTHENTICATION_API_URL")}/";
+            var idTokenValidation = new IdTokenRequirements(issuer, GetVariable("AUTH0_CLIENT_ID"), TimeSpan.FromMinutes(1));
+            var verifier = await AsymmetricSignatureVerifier.ForJwks(issuer);
```

Comment:
Should add tests for symmetric as well.

Reply:
Having some problems with symmetric integration tests so will add that in another PR to avoid more merge problems as other parts land.